Sources/Storyboard/StoryboardSceneBased.swift

```diff
@@ -42,8 +42,8 @@ public extension StoryboardSceneBased where Self: UIViewController {
    - returns: instance of the conforming ViewController
    */
   static func instantiate() -> Self {
-    let storyboard = Self().storyboard
-    guard let vc = storyboard?.instantiateViewController(withIdentifier: self.sceneIdentifier) as? Self else {
```

Comment:
Ah, right. Then shouldn't we use `Self.storyboard` (And not `self.storyboard`) in `instantiate()` then?

I agree that renaming the property to avoid confusion would be a good idea but as it's also a breaking change (and thus requires to bump the major version) I'm trying to avoid it just to solve a bug 😉